Libplanet.Headless/Hosting/LibplanetNodeService.cs

```diff
@@ -363,6 +363,14 @@ Task BootstrapSwarmAsync(int depth)
 
                         Log.Error(e, "{Message}", e.Message);
                     }
+                    catch (Exception e)
+                    {
+                        Log.Error(
+                            e,
+                            "An exception occurred during preload async: {Message}",
```

Comment:
I wonder the message should have `async` description. If you wanted to mention `Swarm.PreloadAsync`, in my opinion, it seems better to use `nameof` keyword. Like...

```suggestion
                            $"An exception occurred during {nameof(Swarm.PreloadAsync)}: {{Message}}",
```

Or without `async` mention...

```suggestion
                            "An exception occurred during preloading: {Message}",
```

-----

Also it seems unexpected exception. So, how about adding the *unexpected* adjective?

```suggestion
                            "An unexpected exception occurred during preload async: {Message}",
```